siff-dr/siff-header.go

```diff
@@ -10,8 +10,8 @@ import (
 
 // SIFF constants
 const (
-	EXP  layers.IPv4Flag = 1 << 3
-	EVIL layers.IPv4Flag = 1 << 2 // http://tools.ietf.org/html/rfc3514 ;)
+	EXP  layers.IPv4Flag = 1 << 2
+	EVIL layers.IPv4Flag = 1 << 3 // http://tools.ietf.org/html/rfc3514 ;)
```

Comment:
> The Go style seems to suggest that constants are all caps.

I think that's just because, in that example, the constants are actually acronyms. [The standard library follows the mixed caps convention](https://golang.org/pkg/net/http/#pkg-constants).

tbh I don't really care if we follow the best programming practices at this point, just thought I'd mention it

> But they're not actually being used in the flags field, I just maintained the conventions to not break the existing usage of the functions. It's actually being used as the position in one of the bytes in the first IPv4 option, used for the flags.


Ok, sounds good.  They should probably just be `uint8`s instead of `layers.IPv4Flag`s.